nora/src/content/browser-splitView/browser-splitView.tsx

```diff
@@ -0,0 +1,215 @@
+import { insert } from "@solid-xul/solid-xul";
+import splitViewStyle from "./browser-splitView.pcss?inline";
+import { HideSplitViewSplitter, SplitViewSplitter } from "./split-view-splitter";
+
+export namespace gSplitView {
+  let StyleElement = () => {
+    return <style id="splitViewCSS">{ splitViewStyle }</style>
+  }
+
+  type tab = {
```

Comment:
please use UpperCamelCase in type
it is to differential between variable and type(class, etc)